### DISCUSSION TITLE
No training log when the num of iterations is smaller than the default interval
修改“当迭代次数小于默认间隔时，不会输出训练日志“的问题

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#321] [MMEngine]
# No training log when the num of iterations is smaller than the default interval
修改“当迭代次数小于默认间隔时，不会输出训练日志“的问题
 

*No training log when the num of iterations is smaller than the default interval. See more details in* [issue#935](issue#935) .

当迭代次数小于默认间隔时，不会输出训练日志。更多详细信息参见问题＃935 [https://github.com/open-mmlab/mmengine/issues/935](https://github.com/open-mmlab/mmengine/issues/935) 

## Task Information

**Repository**：MMEngine

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1046

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by CastleDream** [comment]
This BUG has been solved in <https://github.com/open-mmlab/mmengine/pull/1046>， 
please label the mission as StageDone

**Reply by crazysteeaam** [reply]
done~
